docs/guide-to-triaging.md

```diff
@@ -0,0 +1,158 @@
+
+- Other forums (Wow/MMO-Champion) - the Wow forums can occasionally be useful when they discuss vanilla or TBC issues, but are usually full of people contradicting and flaming each other, so aren't very reliable as sources.
+
+- https://www.mmo-champion.com/forum.php - Similarly, MMO-Champion has forums that can be useful because they have been going, essentially unchanged, since before Wrath. This means you can sometimes find useful firsthand information there. However, like most forums, sorting the useful information from the random chaff can be difficult.
+
+- https://wowgaming.altervista.org/aowow/ - this is a site acting as the front end to a snapshot of our own AC database. As a result, it provides a rapid way of seeing
+
+- https://wotlkdb.com/ - another third-party snapshot of Wrath-era data, although the quality of its data is occasionally questionable. Not a primary source.
+
+- The triaging Discord's #need-help channel is very useful here.
+
+## Tools
+- The AC server/test environment, This is the basic testing tool. Try to keep your AC server as up-to-date as you can manage. Daily updates are a good idea, and you shouldn't go more than a week without updating. 
+
+- GM commands and macros - see https://www.azerothcore.org/wiki/gm-commands At a bare minimum you want to be familiar with the various cheats (no damage, speed, fly), `.go`, `.additem`, `.quest add` and so on. It can also be useful to setup various cheat and boost macros on your test account, so you can instantly setup and train a new character. When testing various class and racial issues you find yourself making and then deleting a lot of different characters, so automating the new character setup with macros can repay the time invested quite quickly. 
+
+- Keira - this is the indispensable tool for triaging. Five minutes with Keira will save an hour of running around in the open world. It comes with a built-in SQL editor that you can use to run custom queries. It also has built-in tooltips on most data fields that can help understand what you're looking at. You can also use it to generate SQL for fixing things, although the SQL it generates should only be used as a starting point. This is because it often does not meet the preferred SQL standards for PRs regarding duplication of queries.
+
+- A generic SQL editor like HeidiSQL is also useful. Keira doesn't access every single table in the AC database, just the most useful ones, so a tool like Heidi that gives you views of everything can be good for more obscure tables.
+
+- [AC Wiki tables](https://www.azerothcore.org/wiki/database-world ), especially those covering the world database. These give you mportant information about the structure and fields of the DB. If you're puzzled about how something should work or what a flag means, the answer is often here.
```

Comment:
use relative links when linking to a wiki page. See https://azerothcore.org/wiki/wiki-standards

Reply:
Fixed.